_locales/en/messages.json

```diff
@@ -2028,7 +2025,12 @@
 		"message": "Priority Support"
 	},
 	"subscribe_pitch": {
-		"message": "While Ghostery is free, you can choose to support us through a small subscription of $$2 per month in exchange for cool perks, such as color themes, priority help service, and more. Join our mission and subscribe!"
+		"message": "While Ghostery is free, you can choose to support us through a small subscription of $DOLLARS$ per month in exchange for cool perks, such as color themes, priority help service, and more. Join our mission and subscribe!",
+		"placeholders": {
+			"dollars": {
+				"content": "$1"
```

Comment:
@christophertino No, it should be $1 because Serge is passing in `$2` into the translation function: `t('subscribe_pitch', '@2')` and $1 in the `placeholder` `content` is for when you will fill that space with a passed variable.

Reply:
@zarembsky There is another escaped `$` in `hub_supporter_price`:
The `$$` in between `$SUP_START$` and `$SUP_END$` is an escaped `$`.
Not sure if all escaped `$` need to be fixed...

Reply:
Yes, same thing.

Reply:
$$ in Hub Supporter tab is fixed. Could not find any more similar cases.